### ISSUE TITLE
APIv2 user.fields response object not included

### ISSUE THREAD

**Issue by factorlive** [geduldig/TwitterAPI#171]
The APIv2 response uses an additional "includes" response object in addition to "data". Therefore, the APIv1.1 approach requires more customization for APIv2.

Endpoint 

`tweets`

Code

```python3
api = TwitterAPI(os.getenv("CONSUMER_KEY"), os.getenv("CONSUMER_SECRET"),
                 os.getenv("TOKEN"), os.getenv("SECRET"), api_version='2')

r = api.request('tweets', {'ids': '1362531221746704385', 'expansions': 'author_id',
                           'user.fields': USER_FIELDS, 'tweet.fields': 'created_at'})
for item in r:
    print(r.json())
    print("------assert equal------")
    print(item)
```

Output

```python3
{'data': [{'created_at': '2021-02-18T22:35:32.000Z', 'author_id': '1332174395561963528', 'id': '1362531221746704385', 'text': '@birdwatch @Twitter @TwitterSupport @TwitterDev'}], 'includes': {'users': [{'pinned_tweet_id': '1362260991539613698', 'username': 'ohthatstevenguy', 'public_metrics': {'followers_count': 190, 'following_count': 1323, 'tweet_count': 7032, 'listed_count': 0}, 'profile_image_url': 'https://pbs.twimg.com/profile_images/1332178860654387200/P_MWls9G_normal.jpg', 'verified': False, 'url': '', 'description': 'Taxes for None\nSlurpees for All\nBro/Brah/Dude\n\nActual Libertarian', 'protected': False, 'created_at': '2020-11-27T04:08:31.000Z', 'id': '1332174395561963528', 'name': 'Oh That Steve! 🏴\u200d☠️🇬🇧🇺🇸🇮🇱 🇨🇦'}]}}
------assert equal------
{'created_at': '2021-02-18T22:35:32.000Z', 'author_id': '1332174395561963528', 'id': '1362531221746704385', 'text': '@birdwatch @Twitter @TwitterSupport @TwitterDev'}
```

**Comment by geduldig** [comment]
Yes, agreed. The "includes" section requires a new strategy. As you demonstrate, the only way to see it is with `r.json()`, and this will not work when paging with `TwitterPager`. 

I'm almost ready to upload a solution to this. You can follow github.com/geduldig/TwitterAPI/issues/156 to see progress. I expect to have something in the next or two. Basically, it will insert the data from the "includes" section back into the tweet in the "data" section.

**Comment by factorlive** [comment]
Ok, cool. Will close this one then.

**Comment by geduldig** [comment]
This feature is now available in v2.6.8.
